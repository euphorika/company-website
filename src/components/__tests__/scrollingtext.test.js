import React from "react"
import renderer from "react-test-renderer"

import ScrollingText from "../scrollingtext"

describe("ScrollingText", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ScrollingText title="Awesome Headline">Awesome Content</ScrollingText>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
