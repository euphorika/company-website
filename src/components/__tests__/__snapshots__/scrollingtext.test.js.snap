// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollingText renders correctly 1`] = `
<section
  className="scrollingTextContainer"
>
  <div
    className="title"
  >
    <h2
      dangerouslySetInnerHTML={
        Object {
          "__html": "Awesome Headline",
        }
      }
    />
  </div>
  <div
    className="body"
  >
    Awesome Content
  </div>
</section>
`;
